ui/app/AppLayouts/WalletV2/components/AddAccount.qml

```diff
@@ -1,24 +1,26 @@
 import QtQuick 2.13
 import QtQuick.Controls 2.13
+import StatusQ.Controls 0.1
 import "../../../../shared"
 import "../../../../shared/status"
 import "../../../../imports"
 
-StatusRoundButton {
+StatusButton {
     id: btnAdd
+    width: 138
+    height: 38
+    text: qsTr("Add account")
+    //TODO icon is not visible
```

Comment:
The problem here is most likely that `StatusButton` in Status Desktop shadows `StatusButton` from `StatusQ.Controls`. If you move the import of `StatusQ.Controls` further down, it should work.

Reply:
Switched to use StatusFlatButton and works like a charm